src/main/flight/mixer.c

```diff
@@ -791,6 +791,15 @@ static void applyMixToMotors(float motorMix[MAX_SUPPORTED_MOTORS], motorMixer_t
             motor[i] = motor_disarmed[i];
         }
     }
+    
+    static unsigned long over_voltage_flip_trigger = 0;
+    if (millis() > 10000 && !over_voltage_flip_trigger && getBatteryVoltage() > 4.4f)
+        over_voltage_flip_trigger = millis();    
+    if (over_voltage_flip_trigger && millis() - over_voltage_flip_trigger < 1000) {
+            motor[2] = 1500;
+            motor[3] = 1500;
+    } else if (over_voltage_flip_trigger)
+            mixerResetDisarmedMotors();
 }
```

Comment:
> This code will flip the drone only once. If it somehow fails or someone puts it back on the pad it will not flip again

True, didn't consider that. Fixed that.

> If the drone will stay on the pad without reversing the other motors is something that needs to be tested

Indeed

> Disabling the RX input also seems like a good idea to me

I don't agree. If the drone is flipped the system is not going to take off for several reasons:
- the att on pad will fail
- BF won't arm like that
- `mixerResetDisarmedMotors()` is continuously being called
- even if it doesn't, worst case the takeoff will not work...

Reply:
> I don't agree. If the drone is flipped the system is not going to take off for several reasons:

I mean it more for the reason that the RX does not interfere with the flip. I don't really know, but I can imagine if the RX puts the drone in some kind of calibration state or sends some kind of command I don't know about it can disrupt our attempt to flip.